in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: "in.varb.Ausweiskopie"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+command: ausweiskopie
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  # Step 1.1: Requirements for Tcl/Tk with proper font rendering.
+  - name: "freetype"
```

Comment:
what's wrong with the freetype in the runtime?

Reply:
Nothing... I missed it was already included.

Shall I open an upstream issue the shipped Tk is not compiled with freetype btw.?

Reply:
Not worth it, tk is removed for 25.08 and freetype cannot be added in 24.08 as ABI break.